index.html

```diff
@@ -473,7 +473,62 @@ <h6 class="title newsletter-title">Subscribe to News Flash</h6>
     </div>
 
   </section>
+<!-- Blog post -->
+<section class="blog-post section-header-offset">
+  <div class="blog-post-container">
+    <div class="blog-post-data">
+      <h3 class="title blog-post-title">Is the VR the future?</h3>
+      <div class="article-data">
+        <span>Dec 6th 2021</span>
+        <span class="article-data"></span>
+        <span>4 min read</span>
+      </div>
+      <img src="./assets/images/feature/feature-1.jpg" alt="">
+    </div>
+    <div class="container">
+      <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor
+      incididunt ut labore et dolore magna
+      aliqua.</p>
 
+      <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor
+        incididunt ut labore et dolore magna
+        aliqua.</p>
+
+        <blockquote class="quote">
+          <p>
+            <span><i class="ri-double-quotes-l"></i></span>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor
+            incididunt ut labore et dolore magna
+            aliqua.<span><i class="double-quotes-r"></i></span>
+          </p>
+        </blockquote>
+        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor
+          incididunt ut labore et dolore magna
+          aliqua.</p>
+    </div>
+    <div class="author d-grid">
+      <div class="author image-box">
+        <img src="./asssets/images/author.jpg" alt="" class="article-image">

```

Comment:
This alt property is empty. Please enter an alternative im image text to be used. Thanks.

[alt Attribute](https://www.w3schools.com/TAGS/att_alt.asp)